modules/acquia_cms_headless/tests/src/Functional/HeadlessBrowserTestBase.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Drupal\Tests\acquia_cms_headless\Functional;
+
+use Acquia\DrupalEnvironmentDetector\AcquiaDrupalEnvironmentDetector;
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Base class for the HeadlessDashboard Browser tests.
+ */
+class HeadlessBrowserTestBase extends BrowserTestBase {
```

Comment:
**HeadlessBrowserTestBase** looks identical to **DashboardBrowserTestBase** class only difference is **DashboardSectionTrait** in the **DashboardBrowserTestBase** class

@deepakmishra2 Can we remove DashboardBrowserTestBase as only class DashboardApiUsersTest extends DashboardBrowserTestBase?
Instead we can extended HeadlessBrowserTestBase and include the trait DashboardSectionTrait within DashboardApiUsersTest?

cc: @vishalkhode1 @rajeshreeputra @chandan-singh7929

Reply:
@apathak18 No we cannot do that because going forward we will be having more headless test [ACMS-1484, ACMS-1481, ACMS-1485, etc] in which we will be using HeadlessBrowserTestBase and not DashboardBrowserTestBase.

Reply:
With the Merge of PR https://github.com/acquia/acquia_cms/pull/1419/files#diff-c3c58804ab9ff9eef2745abf8d76c4975b02062d7527ed69e0eadf58bf778874R11
@deepakmishra2 I guess you need to rebase and remove this file.

cc: @chandan-singh7929 @vishalkhode1